extensions/ext-llm-anthropic/README.md

```
@@ -212,8 +212,34 @@ providerOptions: {
+      "type": "json_schema",
+      "schema": { "type": "object", "properties": {}, "additionalProperties": false }
+    }
+  }
+}
+```
+
+Anthropic rejects an object-typed schema that does not explicitly set
+`additionalProperties: false`, independently of the framework's own `strict`
+flag:
+
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)